routes/sessionRoutes.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express');
+const passport = require("passport");
+const router = express.Router();
+
+const {
+  logonShow,
+  registerShow,
+  registerDo,
+  logoff,
+} = require("../controllers/sessionController");
+
+router.route("/register").get(registerShow).post(registerDo);
+router
+  .route("/logon")
+  .get(logonShow)
+  .post(
+    passport.authenticate("local", {
+      successRedirect: "/",
+      failureRedirect: "sessions/logon",
```

Comment:
Missing leading "/", it should be /sessions/logon

Current code causes a 404 error:
Sorry, /sessions/sessions/logon not found!